src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/masonry-decorator/masonry-view.js

```diff
@@ -90,9 +92,9 @@ define([
                 '       <% if (!!selectable) { %>',
                 '       <div class="footer-checkbox custom-checkbox"><input type="checkbox"><span class="icon"></span></div>',
                 '       <% } %>',
-                '       <a href= "<%= downloadUrl %>" class="fa-cloud-download footer-download ' + constants.downloadNavigatorClass + '"></a>',
+                '       <div class="fa-cloud-download footer-download footer-icon ' + constants.downloadNavigatorClass + '"></div>',
```

Comment:
Ah, because it is a dropdown now... Still breaking accessibility... But nevermind.